sections/Content/TextPlusVideo.tsx

```diff
@@ -0,0 +1,72 @@
+import type { ImageWidget } from "apps/admin/widgets.ts";
+import Image from "apps/website/components/Image.tsx";
+
+
+export interface Props {
+    /**
+     * @title reversed order?
+     */
+    directionInverse?: boolean;
+
+    /**
+    * @title Image of title content
+    * @description Infos of primary column
+    */
+    image: {
+        src: ImageWidget;
+        alt: string;
+    };
+    /**
+     * @title Text content 
+     * @format html 
+     */
+    textContent: string;
+
+    /**
+     * @title List of imagens
+     */
+    listImages: {
+        src: ImageWidget;
+        alt: string;
+    }[];
```

Comment:
Sobre o comentario do Suzano do formulario, dessa forma o que o usuario colocar no `alt` será considerado o nome do item no admin
```suggestion
    listImages: {
        src: ImageWidget;
        /** @title Alt */
        label: string;
    }[];
```